frontend/src/metabase/core/components/Button/Button.tsx

```diff
@@ -12,7 +12,6 @@ const BUTTON_VARIANTS = [
   "large",
   "round",
   "primary",
-  "secondary",
```

Comment:
It seems that `secondary` is still used here https://github.com/metabase/metabase/blob/3f08385d72aa27ce49e08e48292b870d3a2a9889/frontend/src/metabase/reference/components/EditableReferenceHeader.jsx#L84